packages/app/src/vis-packs/core/complex/config.tsx

```diff
@@ -20,7 +20,7 @@ function createComplexConfigStore() {
       }),
       {
         name: 'h5web:complex',
-        version: 2,
+        version: 3,
```

Comment:
The values of the `ComplexVisType` enum have changed, so I bump the versions of the complex configs to avoid issues.